index.html

```diff
@@ -0,0 +1,118 @@
+            <span style="border: solid black; padding:3px; margin:3px">
+                <select onclick="Auto()" class="dropbtn" style='margin-left:3px'>
+                    <option>Left</option>
+                    <option>Right</option>
+                </select>
+                <button>Confirm</button>
+            </span>
+
+                <!--
+                    Modes:
+                    DOuble Switch
+                    Double scale
+
+                    Tutorial:
+                    https://www.youtube.com/watch?v=rUjiGOct808
+                !-->
+                <hr />
+                <span style="border:solid black;padding:2px;margin-left:2%;margin-right:2%">
+                        <!--
+                           Insert Checklist
+                        !-->
+                    <select onclick="Auto()" class="dropbtn" style='margin-left:3px'>Auto Mode
+                            <option>Checklist</option>
+                            <option>///</option>
+                            <option>///</option>
+                            <option>///</option>
+                            <option>///</option>
+                        </select>
+                        <button onclick="IB()">Mark</button>
+                    </span>
+                </div>
+            <!--
+                End of 2nd Box
+            !-->
+            <!-- 
+                Box 3 // Time -Timer -Cycle Times
+            !-->
+            <div style='border:2px solid crimson;padding-top:5px;padding-bottom:5px;margin-top:10px;background-color:white'>
+                <span id="Timr"></span>
+                <hr>
+                <span>Cycle Time</span>
+            </div>
+            <div style='padding:2px;margin:0;background-color:white;padding-top:10px;padding-bottom:15px;border: 2px solid red'>
+                <span style="border: solid black;margin:5px;">Speed</span>
+                <hr>
+                    <!--
+                        Insert Java script for Speed here
+                    !-->
+            </div>
+            <div style='border:2px solid crimson;padding-top:80px;padding-bottom:80px;margin-left:-385%;margin-right:420%;margin-top:-190%;background-color:white'>Speed&Direction
+            </div>
+            <div style='border:2px solid crimson;padding-top:80px;padding-bottom:80px;margin-left:-385%;margin-right:420%;margin-top:-3%;margin-bottom:100%;background-color:white'>Orientation
+            </div>
+        </div>
+        </div>
+        <!-- 
+            End of Right section
+        !-->
+        <div style="border: solid black;padding:10px;margin:10px;background-color:darkgray">
+            <!--
+                Box 1 // Driver Instructions
+            !-->
+            <div>
+                    Driver Instructions
+            </div>
+        </span>
+        <script src="JS/Timer.js"></script>
+    </body>
+</html>
+<script>
+    function IB(){
+        alert('Not complete')
+    }
+</script>
```

Comment:
Please keep all JS in a separate file. Editing inline JS in HTML files is not fun.